app/src/main/java/ch/epfl/sweng/SDP/home/HomeActivity.java

```diff
@@ -57,10 +59,12 @@ public static void disableBackgroundAnimation() {
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+
         overridePendingTransition(0, 0);
         setContentView(R.layout.activity_home);
         profileWindow = new Dialog(this);
 
+
```

Comment:
why ?

Reply:
I did not create this code

Reply:
He meant the empty line I think

Reply:
I know it's not a big mistake but let's keep the code uniform (i.e. no double empty line unless necessary) :D